### DISCUSSION TITLE
Add more actions when clicking on a notification

### DISCUSSION THREAD

**Discussion by yungsamd17** [Chatterino/chatterino2#4910] [Ideas]
## Add more options to Click on a Chatterino: **Live Notification** like:
- Open channel in new tab _(Chatterino new tab)_
- Open stream in browser, and channel in new tab 
- Open player in browser, and channel in new tab

### Examples:

Chatterino settings:

![new-qol-feature](https://github.com/Chatterino/chatterino2/assets/64147848/c87a0fd1-2896-40a3-a5c6-9bbe28d096db)

Desktop Notification:

![Screenshot 2023-10-20 234308](https://github.com/Chatterino/chatterino2/assets/64147848/fd1958ef-01a0-473e-9da3-35bc343e9e59)

If it's not possible to add option of opening steam in browser and channel in chatterino, having option for at least to open channel in new tab in chatterino would be so much more handy. 

I myself have channel I have chatterino notifs on even and don't have channel open, just because they either are not live that often or I just want to have less/cleaner channel tab in chatterino.

**Comment by kornes** [comment]
maybe we could utilize notification actions for that, and offer customization for buttons

![image-actions](https://github.com/Chatterino/chatterino2/assets/28986062/5a8f1094-aaa9-4a6b-b40b-c26cc7562396)
https://github.com/mohabouje/WinToast#actions

**Reply by Nerixyz** [reply]
I thought of this as well. Though, I wasn't sure how to customize it or where to put stuff. For live notifications, there's (mostly) only one action that makes sense for a user. However, I can see this being useful for https://github.com/Chatterino/chatterino2/issues/398 (maybe in conjunction with [a quick reply](https://learn.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/adaptive-interactive-toasts?tabs=appsdk#quick-reply-text-box)).

---

Also, context menu actions might be nice (see https://github.com/Chatterino/chatterino2/discussions/4831):

![](https://github.com/Chatterino/chatterino2/assets/19953266/a6bd7bea-9258-4b78-a7f9-c685be7c5f85)
